include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -258,3 +258,20 @@ def WhileOp : CalyxContainer<"while", [
   ];
```

Comment:
Please add a TODO to the CalyxEmitter [1], and a failure in case somewhere tries to emit an Invoke operation. Or, update the emitter with a test. Up to you.


1: https://github.com/llvm/circt/blob/main/lib/Dialect/Calyx/Export/CalyxEmitter.cpp